llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/ConvertToLoops.cpp

```diff
@@ -82,12 +82,15 @@ struct TilingInterfaceLowerToLoopsPattern : public RewritePattern {
     if (isa<linalg::LinalgOp>(op)) {
       return rewriter.notifyMatchFailure(op, "ignoring LinalgOps");
     }
-    if (llvm::any_of(tilableOp->getResults(),
-                     [&](Value v) { return v.getType().isa<ShapedType>(); })) {
-      return rewriter.notifyMatchFailure(
-          tilableOp, "lower to loops needs to have tensor semantics");
-    }
+    // Tensor semantics?
```

Comment:
historically lower to loops needs to have buffer semantics but we can lift if upstream supports